com.unity.ml-agents/Runtime/Policies/RemotePolicy.cs

```diff
@@ -17,7 +17,9 @@ internal class RemotePolicy : IPolicy
         string m_FullyQualifiedBehaviorName;
         ActionSpec m_ActionSpec;
         ActionBuffers m_LastActionBuffer;
+#if MLA_UNITY_ANALYTICS_MODULE
```

Comment:
Is this necessary for improving coverage?

Reply:
probably a merge mistake